install_config/install/disconnected_install.adoc

```diff
@@ -320,6 +359,42 @@ and then transporting them:
     registry.access.redhat.com/openshift3/metrics-cassandra \
     registry.access.redhat.com/openshift3/metrics-heapster
 ----
++
+----
+# docker save -o ose3-logging-metrics-images.tar \
+    registry.access.redhat.com/openshift3/container-engine \
+    registry.access.redhat.com/openshift3/efs-provisioner \
```

Comment:
just like what I mentioned in my previous comments, some images are put into wrong category.